From 116db28cea0604e812648302670f8d73504e60ea Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sat, 11 Aug 2018 14:48:52 +0300
Subject: [PATCH] hl: make: fix dali-distclean

The defconfig has been parameterized and now depends on the
HL_DEV setting which is set as a dependency on the build target
(e.g. dali-asic-build), but not for the dali-distclean...
The dali-distclean fails with:

boot/uboot/uboot.mk:347: *** No board defconfig name specified, check your BR2_TARGET_UBOOT_BOARD_DEFCONFIG setting.  Stop.
Makefile:79: recipe for target '_all' failed
make[2]: *** [_all] Error 2
make[2]: Leaving directory '/home/jenkins/workspace/tbot/buildroot'
Makefile:41: recipe for target 'uboot-dirclean' failed
make[1]: *** [uboot-dirclean] Error 2
make[1]: Leaving directory '/home/jenkins/workspace/tbot/buildroot-external'
Makefile:137: recipe for target 'dali-distclean' failed
make: *** [dali-distclean] Error 2

Make the dali-distclean depend on the hl-dali taget.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 967034b..7f3846c 100644
--- a/Makefile
+++ b/Makefile
@@ -133,7 +133,7 @@
 endif
 
 .PHONY: dali-distclean
-dali-distclean: $(OUTPUT_DIR)/.config
+dali-distclean: hl-dali $(OUTPUT_DIR)/.config
 	@$(MAKE) uboot-dirclean linux-dirclean armcp-dirclean	\
 		 spdlog-dirclean specs-dirclean specs_external-dirclean \
 		 host-uboot-test-hooks-dirclean host-tbot-dirclean \
